lib/groonga-query-log/command/format-regression-test-logs.rb

```diff
@@ -129,29 +134,29 @@ def report_diff(command, response_old, response_new)
                      "--label=old",
                      "--label=new",
                      "-u",
-                     response_old_file.path, response_new_file.path)
+                     response_old_file.path, response_new_file.path, :out => @output)
```

Comment:
It doesn't work with IO like object such as `StringIO`.
If `StringIO` works, it's easy to write test.

How about this?

```ruby
Tempfile.open("response-diff") do |response_diff_file|
  system("diff",
         "--label=old",
         "--label=new",
         "-u",
         response_old_file.path, response_new_file.path,
         out: response_diff_file)
  response_diff_file.flush
  @output.print(response_diff_file.read)
end
```

Reply:
It's out of scope of this pull request but we should not depend on `diff` command for portability. For example, we can use diff-lcs gem: https://github.com/clear-code/redmine-plugin-wiki-change-notifier/blob/master/init.rb#L15